library/sl

```diff
@@ -8,11 +8,11 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 6f0afe07c9abcb8952c22ab76ed69d1fc757602e
+GitCommit: 4748d2717b329e8516d5ddedff8de459b1e8e20c
```

Comment:
Sorry, this one should be `0d8792816e36ec1a2f2023f48858737f6d683680` now, right? (https://github.com/scientificlinux/sl-docker/commit/0d8792816e36ec1a2f2023f48858737f6d683680 from https://github.com/scientificlinux/sl-docker/tree/7)

Reply:
I really should stop doing two things at once.... fixed...